charts/hono/requirements.yaml

```diff
@@ -13,11 +13,11 @@
 dependencies:
   - name: prometheus
     version: ~11.7.0
-    repository: "https://kubernetes-charts.storage.googleapis.com/"
+    repository: "https://charts.helm.sh/stable"
     condition: prometheus.createInstance
   - name: grafana
     version: ~3.8.15
-    repository: "https://kubernetes-charts.storage.googleapis.com/"
+    repository: "https://charts.helm.sh/stable"
```

Comment:
The Grafana chart has moved as well (see [here](https://github.com/helm/charts/tree/master/stable/grafana)).

So we could use `https://grafana.github.io/helm-charts` here instead already.